packages/core/src/runner/helper.ts

```diff
@@ -37,29 +39,29 @@ export function getLog(pipcookRunner: PipcookRunner): any {
  * @param self: the target update runner
  * @param saveModelCallback
  */
-export async function assignLatestResult(updatedType: string, result: any, self: PipcookRunner, saveModelCallback?: Function) {
+export async function assignLatestResult(updatedType: string, result: PipcookComponentOutput, self: PipcookRunner, saveModelCallback?: Function) {
   switch (updatedType) {
   case DATA:
-    self.latestSampleData = result;
+    self.latestSampleData = result as UniDataset;
     break;
   case MODEL:
-    self.latestModel = result;
+    self.latestModel = result as UniModel;
     break;
   case EVALUATE:
     console.log('evaluate result: ', result);
```

Comment:
May console.log should be replaced?

Reply:
A TODO is to be added here, It is retained here because there is no ability to obtain the value of evaluate from the outside, so we need to use it to confirm whether the test passed, but we can also use `debug` instead, but I recommend that we complete it in another PR.

Reply:
Approve:)